src/app/user/profile/profile.page.ts

```diff
@@ -10,12 +11,18 @@ import { AuthService } from 'src/shared/services/auth.service';
   styleUrls: ['./profile.page.scss'],
 })
 export class ProfilePage implements OnInit {
-  user: any = {email: '', firstName: '', lastName: '', region: ''};
+  user: User | null = {
+    email: '', firstName: '', lastName: '', region: '',
+    role: '',
+    organizatorName: '',
```

Comment:
If it isn’t an organizer, there shouldn’t be any organizatorName. You don’t have to pass an empty Uset object with all properties, if it’s the main goal here

Reply:
In the update function, I will check what the user is and submit only what I need.